packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -59,43 +57,49 @@ interface CoinjoinStatusBarProps {
 
 export const CoinjoinStatusBar = ({ accountKey, session, isSingle }: CoinjoinStatusBarProps) => {
     const devices = useSelector(state => state.devices);
-    const accounts = useSelector(state => state.wallet.accounts);
+    const relatedAccount = useSelector(state => selectAccountByKey(state, accountKey));
     const selectedDevice = useSelector(state => state.suite.device);
-    const routerParams = useSelector(state => state.router.params);
+    const routerParams = useSelector(selectRouterParams);
     const discovery = useSelector(state => state.wallet.discovery);
 
-    const { goto, selectDevice } = useActions({
-        goto: routerActions.goto,
-        selectDevice: suiteActions.selectDevice,
-    });
+    //  ⬇️⬇️⬇️ REMOVE AFTER REVIEW ⬇️⬇️⬇️
+
+    // will be replaced by an condition in the onCoinjoinClientEvent() action, with the same functionality
+    // https://github.com/trezor/trezor-suite/blob/1be74a2f30b65f4c0aa1567e7e6208f78410256b/packages/suite/src/actions/wallet/coinjoinClientActions.ts#L135
+
+    // useCoinjoinTracker(accountKey, session, relatedDevice);
```

Comment:
I implemented the functionality before realising that the statuses are tracked by actions on the coinjoin branch already. Decided to leave it basically as pseudocode to have some context for the modals _(although the hook is functional)_